tests/lib_tests.rs

```diff
@@ -12,7 +12,7 @@ fn major_required_bump_if_breaking_change() {
     let (_crate_name, crate_report) = report.crate_reports().iter().next().unwrap();
     let required_bump = crate_report.required_bump().unwrap();
     assert_eq!(required_bump, ReleaseType::Major);
-    assert_eq!(crate_report.detected_bump(), ActualSemverUpdate::NotChanged);
+    assert_eq!(crate_report.detected_bump(), ActualSemverUpdate::Minor);
```

Comment:
```suggestion
    // The "old" and "new" crates have the same version.
    // The detected bump is the minimum-possible SemVer bump in a new release.
    // Since the crates are v0.1.0, the minimum possible bump is minor.
    assert_eq!(crate_report.detected_bump(), ActualSemverUpdate::Minor);
```